test-app/src/main.less

```diff
@@ -1,16 +1,25 @@
 @import url('../../src/styles/main.less');
 
-// Font override:
+// Styles for OMA:
+@import url('./bundles/site-logo.less');
 @import url('./bundles/lato-font.less');
+@import url('./bundles/page-header.less');
+@import url('./bundles/page-footer.less');
+@import url('./bundles/page-template.less');
 @import url('./bundles/home-page-template.less');
+@import url('./bundles/home-page-illustration.less');
 
-@testapp-primary-color: #0d6868;
+@oma-blue-1: #F2F5F8;
+@oma-blue-2: #4F608E;
+@oma-orange-1: #F5E3D6;
+@oma-orange-2: #EAA58D;
+@oma-primary-color: #6D8BB1;
```

Comment:
Just a thought: maybe we could use neutral names like `@theme-color-blue` or similar (not having `oma` in the name). That would make copying easier, and we could have only `@edu-...` and non-`@edu-...` (= local) variables. Not a big issue, we can also leave it like it is or discuss at a later point.

Reply:
With the color palette changes that we do next, these variables will go away and the colors will be directly in educandu. But good point, we should do that in ELMU actually.